lib/src/_constants.dart

```diff
@@ -101,9 +101,11 @@ abstract class ArgumentName {
   static const String timeoutMillis = "timeout";
   static const String emitCurrentValue = "emitCurrentValue";
 
+  static const String uuidsMap = "uuidsMap";
```

Comment:
So why did we decide not to use this map but instead pass in 1 service and an array of characteristics?

Reply:
oh I forgot to remove this. I put the logic in security package, since lantern is like a wrapper. Do you think it's good?

Reply:
Eh, not super thrilled with this API to be honest. This is fine.